CentOS-8 build failing on CI

See log at https://github.com/SymbiFlow/make-env/actions/runs/3778151316/jobs/6422389396
```
Run yum update -y
Error: Failed to download metadata for repo 'appstream': Cannot prepare internal mirrorlist: No URLs in mirrorlist
CentOS Linux 8 - AppStream                      147  B/s |  38  B     00:00    
Error: Process completed with exit code 1.
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -156,7 +156,7 @@ jobs:
           - 7
           - stream8
           - stream9
-    container: centos:${{ matrix.os }}
+    container: quay.io/centos/centos:${{ matrix.os }}
     steps:
 
     - name: Install dependencies

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -144,7 +144,8 @@ jobs:
         cd test
         make test-command
 
-
+# https://en.wikipedia.org/wiki/CentOS#CentOS_releases
+# https://www.centos.org/centos-stream/
   centos:
     runs-on: ubuntu-latest
     strategy:
@@ -153,7 +154,8 @@ jobs:
       matrix:
         os:
           - 7
-          - 8
+          - stream8
+          - stream9
     container: centos:${{ matrix.os }}
     steps:
 

